legend-engine-language-pure-dsl-service/src/main/java/org/finos/legend/engine/language/pure/dsl/service/grammar/from/ServiceParseTreeWalker.java

```diff
@@ -94,6 +95,9 @@ public Service visitService(ServiceParserGrammar.ServiceContext ctx)
         // owners (optional)
         ServiceParserGrammar.ServiceOwnersContext ownersContext = PureGrammarParserUtility.validateAndExtractOptionalField(ctx.serviceOwners(), "owners", service.sourceInformation);
         service.owners = ownersContext != null && ownersContext.STRING() != null ? ListIterate.collect(ownersContext.STRING(), ownerCtx -> PureGrammarParserUtility.fromGrammarString(ownerCtx.getText(), true)) : new ArrayList<>();
+        // tags (optional)
+        ServiceParserGrammar.ServiceTagsContext serviceTagsContext = PureGrammarParserUtility.validateAndExtractOptionalField(ctx.serviceTags(), "tags", service.sourceInformation);
+        service.tags = serviceTagsContext != null && serviceTagsContext.SERVICE_TAGS() != null ? ListIterate.collect(serviceTagsContext.tagDescription(), this::visitServiceTagDescription) : new ArrayList<>();
```

Comment:
can we keep service.tags as `null` not `new ArrayList<>()` when tagsContext are not defined. 
small thing but this would prevent us from returning this property in the Json when we go from grammar to json.

Reply:
change updated